docs/lib/restapi/fragments/flutter/getting-started.md

```diff
@@ -0,0 +1,52 @@
+The Amplify CLI deploys REST APIs and handlers using [Amazon API Gateway](http://docs.aws.amazon.com/apigateway/latest/developerguide/) and [AWS Lambda](http://docs.aws.amazon.com/lambda/latest/dg/).
+
+The API category will perform SDK code generation which when used with the `AWSMobileClient` can be used for creating signed requests for Amazon API Gateway when the service Authorization is set to `AWS_IAM` or when using a [Cognito User Pools Authorizer](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-integrate-with-cognito.html).
+
+See the authentication section for more details for using the `AWSMobileClient` in your application.
+
+## Create new REST API
+
+In a terminal window, navigate to your project folder (the folder that contains your app `.Android Studioproj` file), and add the SDK to your app.
```

Comment:
Does it not need to be changed?

Reply:
thanks updated to: 


In a terminal window, navigate to your project folder (the folder that contains your app `/lib` folder), and add the SDK to your app.